libs/libarchfpga/src/logic_types.h

```diff
@@ -47,11 +57,204 @@ struct t_model {
     void* instances = nullptr;
```

Comment:
Comment these two things are only used by parmys: possible TODO: clean up instances and used.

Reply:
Could put a comment above t_model.  Could be even better to make a Doxygen comment for it.
Moves us toward building doxygen documentation for it.